examples/README.md

```diff
@@ -1 +1,9 @@
 [![Analytics](https://cloud-tools-for-java-metrics.appspot.com/UA-121724379-2/examples)](https://github.com/igrigorik/ga-beacon)
+
+### Simple Example 
+
+You can find simple usage for java in [helloworld](helloworld)
+
+### SpringBoot Example 
```

Comment:
Since this will be in draft mode to start, let's place this in an HTML `<!-- -->` comment for now